src/libraries/Common/src/System/Drawing/KnownColor.cs

```diff
@@ -197,6 +197,9 @@ public enum KnownColor
         MenuHighlight,
 
         // "Web" colors, Part 2
+        /// <summary>
+        /// A system defined color representing the ARGB value <c>#663399</c>.
```

Comment:
Looking at the existing docs, most of these seem to just be "A system-defined color.".  I don't have a strong opinion, but calling out what seems like a discrepancy (if I'm looking in the right place).

Reply:
I saw that some contained the ARGB code, i.e: `Purple`. That's why I added it, but I don't have a strong opinion either.

Reply:
I will merge this PR as is and if you have a strong opinion I can submit a new PR to fix that.